releases/development/master/extra/barclamp_install.rb

```diff
@@ -35,6 +35,17 @@ def usage()
   exit
 end
 
+def naviational_barclamps
+  [
+    "cisco-ucs"
+    "crowbar",
+    "network",
+    "openstack",
+    "swift",
+    "tempest",
```

Comment:
By check, do you mean `if bc[:nav] && bc[:version] < 1` ? Wouldn't that miss the barclamps that had navigation before, but not the nav key?